frontends/web/src/components/wait-dialog/wait-dialog.tsx

```diff
@@ -139,7 +139,7 @@ class WaitDialog extends Component<Props, State> {
         className={style.overlay}
         ref={this.overlay}
         style={{ zIndex: 10001 }}>
-        <div className={style.modal} ref={this.modal}>
+        <div className={[style.modal, style.open].join(' ')} ref={this.modal}>
```

Comment:
Not the regression we talked about - we didn't notice this before (or at least I didn't). 

Since wait-dialog uses CSS of dialog.css, the wait dialog won't visually show on mobile (as it has `transform: translateY(100vh);`). We need to give `transfrom: translateY(0);` so it shows up. To do it we need to add `.open` class as well. Here's a preview of it:

[Before]:

https://user-images.githubusercontent.com/28676406/215666892-812d7533-15cf-4d2d-82fa-5437ceb68023.mp4


[After]:

https://user-images.githubusercontent.com/28676406/215666922-9256684b-eb2c-4002-844a-66152d9c1217.mov



Note: In the `Dialog`, there's a minor delay while adding the `.open` so there's a visible animation. To have the same effect here in `WaitDialog`, I believe we need this to be controlled by its parent (just like `Dialog`) - but this is a bigger topic. So prob. better to do it in a different PR if we really need it.